src/components/Navigator/BaseNavigatorCard.vue

```diff
@@ -0,0 +1,237 @@
+<!--
+  This source file is part of the Swift.org open source project
+
+  Copyright (c) 2022 Apple Inc. and the Swift project authors
+  Licensed under Apache License v2.0 with Runtime Library Exception
+
+  See https://swift.org/LICENSE.txt for license information
+  See https://swift.org/CONTRIBUTORS.txt for Swift project authors
+-->
+
+<template>
+  <div class="navigator-card">
+    <div class="navigator-card-full-height">
+      <div class="navigator-card-inner">
+        <div class="head-wrapper">
+          <div class="head-inner">
+            <button
+              aria-label="Close documentation navigator"
+              :id="SIDEBAR_HIDE_BUTTON_ID"
+              class="close-card"
+              :class="{ 'hide-on-large': !allowHiding }"
+              @click="handleHideClick"
+            >
+              <SidenavIcon class="icon-inline close-icon" />
+            </button>
+            <slot name="head" className="navigator-head" id="INDEX_ROOT_KEY" />
+          </div>
+        </div>
+        <slot name="post-head" />
+        <slot name="body" className="card-body"/>
+        <slot name="filter"/>
```

Comment:
we dont need 3 slots one after the other. One slot is fine imho

Reply:
We may want to separate post-head content from body content, to make sure we don't overwrite the body when adding a post-head.

But yes, body and filter can be the same. I changed it.